### ISSUE TITLE
Alzheimers Detection

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#59]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Alzheimers Detection</br>
:red_circle: **Aim** : Create a image processing model using the DL methods to idenitfy the Alzheimer's affected patient from the MRI images.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/tourist55/alzheimers-dataset-4-class-of-images</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by srini047** [comment]
Hey, @abhisheks008 I would like to work on this issue.

**Full Name:** Sriniketh J
**Github Profile Link:** [srini047](https://github.com/srini047)
**Email ID:** srinikethcr7@gmail.com
**Approach:** Build a model on the dataset and then predict the class of the image provided as (1/4) classes.
**Participant Role:** SSOC-2022

**Comment by sankalp-srivastava** [comment]
@srini047 issue assigned to you. Play around different algorithms and hyperparameter tuning. If you need any help, feel free to reach out.

**Comment by srini047** [comment]
> @srini047 issue assigned to you. Play around with different algorithms and hyperparameter tuning. If you need any help, feel free to reach out.

Sure.

**Comment by abhisheks008** [comment]
@srini047 updates please.

**Comment by srini047** [comment]
@abhisheks008 
I had my placement classes going in my college till 8. Also, I am learning a lot about Tensorflow while doing this. Can you please provide sometime (within 3 days) and I would make a PR.

**Comment by abhisheks008** [comment]
It's okay, thanks for the update @srini047

**Comment by abhisheks008** [comment]
Updates please @srini047

**Comment by abhisheks008** [comment]
@sankalp-srivastava updates please.

**Comment by sankalp-srivastava** [comment]
@srini047 any updates?

**Comment by srini047** [comment]
I will make a PR by Monday. Working on the model.

**Comment by shubham0730** [comment]
Hello @abhisheks008 I would like to work on this issue
Full Name: Shubham Shinde
Github Profile Link: https://github.com/shubham0730
Email ID: shubhamss0730@gmail.com
Approach: Build an ANN and CNN model for prediction
Participant Role: SWOC 2023

**Comment by abhisheks008** [comment]
Issue assigned to you @shubham0730

**Comment by shubham0730** [comment]
Can I upload a kaggle notebook? Because the code isn't running on my pc

**Comment by abhisheks008** [comment]
Download the Kaggle notebook as .ipynb file and upload that.
